### PR TITLE
mvp of rebuilding near.org/blog as BOS components 

### PR DESCRIPTION
**Pull request by jackson-harris-iii**

### CODE REVIEW

File: src/Blog/Feed.jsx [hunk 1]
```diff
@@ -0,0 +1,290 @@
...
+
+const [postToShow, setPostToShow] = useState(null);
+
+const renderItem = (item, index) => {
+  let content;
+  if (item.content.type) {
+    content = item.content;
+  } else {
+    content = item.content;
+    content = JSON.parse(item.content);
```
Comment (charleslavon) [inline src/Blog/Feed.jsx:238]: can be simplified to ```if (!item.content.type)...```

File: src/Blog/Feed.jsx [hunk 2]
```diff
@@ -0,0 +1,290 @@
...
+  });
+}
+
+const getBlogPosts = () => {
+  fetchGraphQL(postsQuery, "FeedQuery", {}).then((result) => {
+    if (result.status === 200) {
+      if (result.body.data) {
+        const posts = result.body.data.dataplatform_near_feed_moderated_posts;
+        setPosts(posts);
+        console, log("ln 44", posts);
```
Comment (shelegdmitriy) [inline src/Blog/Feed.jsx:52]: Or let's fix this 
```suggestion
        console.log("ln 44", posts);
```

**Comment by charleslavon** [comment]
@jackson-harris-iii this MVP UI is looking great.

**Comment by jackson-harris-iii** [comment]
> accountId=jacksonthedev.near&blockHeight=113254562

I went ahead and refactored the code using the PagePost pattern, which greatly reduced the number of pages, and allowed for the use of a preexisting widget to display the markdown. Ground work is in place for adding things like comments, likes, etc. as well.

File: src/Blog/Feed.jsx [hunk 3]
```diff
@@ -0,0 +1,254 @@
...
+
+  const time = new Date(item.block_timestamp / 1000000);
+  const formattedDate = time.toLocaleDateString();
+
+  if (content.type !== "md") {
+    return null;
+  }
+
+  return (
+    <Link href={`/${REPL_ACCOUNT}/widget/BlogPostPage?accountId=${item.account_id}&blockHeight=${item.block_height}`}>
```
Comment (charleslavon) [inline src/Blog/Feed.jsx:230]:
```suggestion
    <Link href={`/${REPL_ACCOUNT}/widget/PostPage?accountId=${item.account_id}&blockHeight=${item.block_height}`}>
```

**Review by charleslavon — APPROVED** [review]

**Pull request merged**